content/docs/gke/deploy/monitor-iap-setup.md

```diff
@@ -71,90 +80,14 @@ problems:
-
-      apiVersion: certmanager.k8s.io/v1alpha1
-      kind: Certificate
-      metadata:
-        creationTimestamp: 2019-04-02T22:49:43Z
-        generation: 1
-        labels:
-          kustomize.component: iap-ingress
-        name: envoy-ingress-tls
-        namespace: istio-system
-        resourceVersion: "4803"
-        selfLink: /apis/certmanager.k8s.io/v1alpha1/namespaces/kubeflow/certificates/envoy-ingress-tls
-        uid: 9b137b29-5599-11e9-a223-42010a8e020c
-      spec:
-        acme:
-          config:
-          - domains:
-            - mykubeflow.endpoints.myproject.cloud.goog
-            http01:
-              ingress: envoy-ingress
-        commonName: kf-vmaster-n01.endpoints.kubeflow-ci-deployment.cloud.goog
-        dnsNames:
-        - mykubeflow.endpoints.myproject.cloud.goog
-        issuerRef:
-          kind: ClusterIssuer
-          name: letsencrypt-prod
-        secretName: envoy-ingress-tls
-      status:
-        acme:
-          order:
-            url: https://acme-v02.api.letsencrypt.org/acme/order/54483154/382580193
-        conditions:
-        - lastTransitionTime: 2019-04-02T23:00:28Z
-          message: Certificate issued successfully
-          reason: CertIssued
-          status: "True"
-          type: Ready
-        - lastTransitionTime: null
-          message: Order validated
-          reason: OrderValidated
-          status: "False"
-          type: ValidateFailed
-      ```
-
-    It can take around 10 minutes to provision a certificate after the 
-    creation of the load balancer.
-
-    The most recent condition should be `Certificate issued successfully`.
-
-    The most common error is running out of [Let's Encrypt 
-    quota](https://letsencrypt.org/docs/rate-limits/).
-    Let's Encrypt enforces a quota of 5 duplicate certificates per week.
-      
-    The easiest fix to quota issues is to pick a different hostname by 
-    recreating and redeploying Kubeflow with a different
-    name. 
-
-    For example if you originally ran the following commands to deploy Kubeflow:
-
-    ```
-    export KF_NAME=my-app
-    export BASE_DIR=<path to a base directory>
-    export KF_DIR=${BASE_DIR}/${KF_NAME}
-    mkdir -p ${KF_DIR}
-    cd ${KF_DIR}
-    kfctl apply -V -f ${CONFIG_FILE}
-    ```
-
-    Then rerun the commands with a different name that you haven't used
-    before:
+1. Verify that a managed certificate resource is generated:
+   
+     ```
+     kubectl describe -n istio-system managedcertificate gke-certificate
+     ```
 
-    ```
-    export KF_NAME=my-app-unique
-    export BASE_DIR=<path to a base directory>
-    export KF_DIR=${BASE_DIR}/${KF_NAME}
-    mkdir -p ${KF_DIR}
-    cd ${KF_DIR}
-    kfctl apply -V -f ${CONFIG_FILE}
-    ```
+     The status field should have information about the current status of Certificate.
+     Eventually, certificate status should be `Active`.
 
 1. Wait for the load balancer to report the back ends as healthy:
```

Comment:
Can you add instructions for checking if the backend is healthy? You can basically copy these instructions
https://www.kubeflow.org/docs/gke/troubleshooting-gke/#502-server-error

Reply:
added reference to the page